docfiles/apptracking.html

```diff
@@ -1,14 +1,16 @@
 <script type="text/javascript">
     window.loadAppInsights = function (includeCookie, telemetryInitializer) {
+        //Backend will patch @id@ and the check would fail for string concat
+        var isProduction = includeCookie && ("@id@" !== "@"+"id"+"@");
```

Comment:
I'm not certain this works in the backend currently; my understanding is that the backend only runs replacements once / on the docfile, and this one is `@included` so it might not get the replacement on `@id@`. That's one that we've run into with docs / share page changes before. I can point you at how to test it on staging though so probably fine to merge and test at least?

Reply:
you should be able to test it by simply doing an uploadtrg

Reply:
Oh right, this is the one that is used in `webapp/public` so this one is testable -- I guess this comment applies more to `docfiles/tracking.html` which is used in other docfiles. Not certain if it goes through the same path though

Reply:
I modified the I tried uploadtrg but the cli kept changing this with local values. Probably uploadtrg handles the html modificiations. I am going to debug later. For now I am going merge and bump and test to see it works. 
https://arcade.makecode.com/app/20f5f48d56153ea3ba8ae89a66761254161d37ed-22a00706f2

Debuging this is hard.